crates/net/Cargo.toml

```diff
@@ -37,7 +38,7 @@ wasm-bindgen-test = "0.3"
 futures = "0.3"
 
 [features]
-default = ["json", "websocket", "http"]
+default = ["json", "websocket", "http", "eventsource"]
```

Comment:
Generally I am reluctant to putting "too" many features into the default one, but as we already have websocket here I guess we can also have eventsource.

Reply:
Yeah, seemed to make sense to continue following the convention.